dockerfiles/force-readthedocs-addons.js

```diff
@@ -73,6 +74,17 @@ async function handleRequest(request) {
   console.log(`ContentType: ${contentType}`);
   console.log(`X-RTD-Force-Addons: ${forceAddons}`);
   console.log(`X-RTD-Hosting-Integrations: ${injectHostingIntegrations}`);
+  console.log(`ContentLength: ${contentLength}`);
+
+  // skip executing the Cloudflare Worker if the response's body is bigger than 1Mb
+  //
+  // the user will see the page rendering properly, but without any of the addons injected
+  // this is an intermediate workaround/solution for
+  // https://github.com/readthedocs/readthedocs-ops/issues/1513
+  if (contentLength && contentLength >= 1000000) {
```

Comment:
contentLenght is probably a string here, if this is working is because of JS implicit type conversion. But we should cast it as integer.